src/SDK/Trace/NoopSpanBuilder.php

```diff
@@ -12,24 +12,20 @@
 
 final class NoopSpanBuilder implements SpanBuilderInterface
 {
-    private static ?self $instance = null;
-
-    public static function getInstance(): self
-    {
-        if (null === self::$instance) {
-            self::$instance = new self();
-        }
-
-        return self::$instance;
-    }
+    /** @var Context|false|null */
+    private $parent;
```

Comment:
wouldn't nullable + typed be better here? eg `private ?Context $parent` ? What does `false` mean?

Reply:
`false` was used when no parent should be used, now setting an empty context on `::setNoParent()` instead.